src/main/resources/META-INF/rewrite/spring-framework-60.yml

```diff
@@ -31,7 +31,17 @@ recipeList:
   - org.openrewrite.apache.httpclient5.UpgradeApacheHttpClient_5
   - org.openrewrite.java.spring.framework.HttpComponentsClientHttpRequestFactoryReadTimeout
   - org.openrewrite.java.spring.framework.MigrateResponseEntityExceptionHandlerHttpStatusToHttpStatusCode
-
+  - org.openrewrite.java.spring.framework.MigrateResponseStatusException
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.framework.MigrateResponseStatusException
+displayName: Migrate breaking changes in `ResponseStatusException`
+description: Migrate Spring Framework 5.3's `ResponseStatusException` method `getRawStatusCode()` to Spring Framework 6's `getStatusCode().value()` and `ResponseStatusException` method `getStatus()` to Spring Framework 6's `getStatusCode()` .
+recipeList:
+  - org.openrewrite.java.spring.framework.MigrateResponseStatusExceptionGetRawStatusCodeMethod
```

Comment:
Perhaps once the recipe for `getStatus` to `getStatusCode` is in place these could be included below line 33, or the recipe on line 33 could be pulled into here and this sub recipe could be transformed into a `HttpStatus` to `HttpStatusCode` recipe